app/components/tech-icons/BunIcon.tsx

```
@@ -0,0 +1,62 @@
+const BunIcon: React.FC<TechIconProps> = ({ className = "", ...props }) => {
+  return (
+    <>
+      <svg
+        className={`hidden sm:block ${className}`}
+        width={47}
+        height={47}
+        viewBox="0 0 80 80"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+        {...props}
+      >
```

Biome (2.4.4)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)